packages/varlet-ui/src/code/Code.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div :class="n()">
+    <div :class="n('content')" v-html="highlightedCode"></div>
```

Comment:
Whether it is possible to remove content blocks ?

Reply:
of course